tasks/concatManifests.js

```diff
@@ -46,6 +46,9 @@ function renderDocumentation(fileName) {
 		};
 
 		out.useCases = find.fileSync(/.*\.json/, paths.useCasesDir)
+			.filter(function (file) {
+				return file.match(/[.]json$/);
+			})
```

Comment:
Wouldn't regexp above take cake of including only .json files? 
Do we need filter in case some other files lead through?